internal/provider/harness_k3s_resource.go

```diff
@@ -285,7 +273,7 @@ func (r *HarnessK3sResource) Create(ctx context.Context, req resource.CreateRequ
 			}
 		}
 
-		resp.Diagnostics.AddWarning("Using k3s harness dev mode, a single (random) k3s harness is exposed to the host and accessible via the kubeconfig file. This works best if only a single k3s harness is created.",
+		log.Warn(ctx, "Using k3s harness dev mode, a single (random) k3s harness is exposed to the host and accessible via the kubeconfig file. This works best if only a single k3s harness is created.",
```

Comment:
Is the goal to not have that be raised to the end user anymore?

Reply:
good call. I changed the relevant warnings back to diagnostics and plumbed the diagnostics through to the response. turns out it was already set up like this 🤦